e107_handlers/form_handler.php

```diff
@@ -867,7 +867,7 @@ private function mediaUrl($category = '', $label = '', $tagid='', $extras=null)
 	{
 		
 		$cat = ($category) ? '&amp;for='.$category : "";
-		if(!$label) $label = ' Upload an image or file';
+		if(!$label) $label = IMALAN_166;
 		if($tagid) $cat .= '&amp;tagid='.$tagid; 
 		

```

Comment:
We can't use IMALAN_XXXX in the form-handler at all. Form-handler can be used on the frontend or backend of e107.